test-cypress/steps/setup/acl-management-steps.js

```diff
@@ -165,6 +169,22 @@ export class AclManagementSteps {
         return cy.get('div.small');
     }
 
+    static getPrefixWarning() {
+        return cy.get('.prefix-warning-text');
+    }
+
+    static getWarningIcon() {
+        return cy.get('.icon-warning');
+    }
+
+    static getWarningIconTooltipText() {
+        return cy.get('.angular-tooltip');
+    }
+
+    static mouseoverWarningIcon() {
+        cy.get('.icon-warning').first().trigger('mouseover');
```

Comment:
This is what I meant with the above comment. You wouldn't needed the `first()` call if you scoped the selector in the first place. Not to mention that you duplicated the `.icon-warning` selector violating the purpose of the step files...